package.json

```diff
@@ -4,38 +4,39 @@
   "version": "0.2.0",
   "description": "Build tooling for LabkiPackManager frontend",
   "scripts": {
-    "build": "rollup -c",
-    "watch": "rollup -c -w",
+    "build": "vite build",
+    "watch": "vite build --watch",
     "test": "vitest run",
     "test:ui": "vitest --ui",
-    "lint": "eslint resources/src --ext .js,.vue",
-    "format": "prettier -w resources/src"
+    "lint": "eslint resources/src",
+    "lint:fix": "npm run lint -- --fix",
+    "format": "prettier -w resources/src",
+    "format:check": "prettier -c resources/src"
   },
   "devDependencies": {
     "@eslint/js": "^9.12.0",
-    "@rollup/plugin-commonjs": "^25.0.8",
-    "@rollup/plugin-node-resolve": "^15.3.1",
-    "@rollup/plugin-terser": "^0.4.4",
-    "@rollup/plugin-typescript": "^12.1.1",
+    "@typescript-eslint/parser": "^8.46.3",
     "@vitejs/plugin-vue": "^5.2.4",
     "@vue/compiler-sfc": "^3.5.22",
     "@vue/test-utils": "^2.4.6",
+    "@vue/tsconfig": "^0.8.1",
     "@wikimedia/codex": "^2.3.2",
     "eslint": "^9.12.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-vue": "^9.28.0",
     "jsdom": "^27.0.0",
+    "mermaid": "^10.9.3",
```

Comment:
added this for the types, but it shouldn't be in the compiled bundle (i think? it's only imported in the type declaration file to assign it to `window`